codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AwsDependency.java

```diff
@@ -55,7 +55,10 @@
     XML_BUILDER(NORMAL_DEPENDENCY, "@aws-sdk/xml-builder", "^1.0.0-beta.1"),
     XML_PARSER(NORMAL_DEPENDENCY, "fast-xml-parser", "^3.16.0"),
     UUID_GENERATOR(NORMAL_DEPENDENCY, "uuid", "^7.0.0"),
-    UUID_GENERATOR_TYPES(DEV_DEPENDENCY, "@types/uuid", "^7.0.0");
+    UUID_GENERATOR_TYPES(DEV_DEPENDENCY, "@types/uuid", "^7.0.0"),
+    MIDDLEWARE_EVENTSTREAM(NORMAL_DEPENDENCY, "@aws-sdk/middleware-eventstream", "^1.0.0-beta.0"),
+    AWS_SDK_EVENTSTREAM_HANDLER_NODE(NORMAL_DEPENDENCY, "@aws-sdk/eventstream-handler-node", "^1.0.0-beta.0"),
+    TRANSCRIBESTREAMING_MIDDLEWARE(NORMAL_DEPENDENCY, "@aws-sdk/eventstream-handler", "^1.0.0-beta.0");
```

Comment:
This dependency enum name doesn't look like it is used. It also doesn't seem to match its name to the package name.